Made new functions in Deck.cs

jacobwmorgan opened the pull request.

jacobwmorgan committed:
Made new functions in deck.cs

jacobwmorgan committed:
Fixed issue in program.cs

cooperj commented on Deck.cs:
```diff
@@ -33,5 +33,26 @@ public bool isEmpty()
             }
             return false;
         }
+        public void shuffle()
```
I would recommend having `deck` passed when calling the function...

cooperj commented on Deck.cs:
```diff
@@ -33,5 +33,26 @@ public bool isEmpty()
+            Console.WriteLine("\nDeck Shuffled");
+        }
+        public void deal()
+        {
```
Dealing function Looks Good!

cooperj commented on Program.cs:
```diff
@@ -14,14 +14,15 @@ static void Main(string[] args)
                     case "":
                         deck.newDeck();
-                        //deck.shuffle();
+                        deck.shuffle();
```
LGTM 👍

cooperj reviewed (approved):
LGTM 👍

cooperj committed:
Merge branch 'main' into review

cooperj committed:
Fixed merge conflict
Added back in the README file after completing a review and pressing the
button which fixed the file, but lost the contents of the file.

cooperj commented on Program.cs:
```diff
@@ -14,14 +14,15 @@ static void Main(string[] args)
+                }
         }
     }
 }
```
You might be missing a bracket at the end
